### PR TITLE
Use spec-up with new features

### PR DESCRIPTION
**Pull request by brianorwhatever**
Switches the project to use my pull request of spec-up

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "merge-stream": "2.0.0",
     "pkg-dir": "4.2.0",
     "prismjs": ">=1.24.0",
-    "spec-up": "^0.10.6",
+    "spec-up": "github:brianorwhatever/spec-up#fix-refs",
```
Comment (m00sey) [inline package.json:58]: we have a newly forked version of spec-up under toip, would this be a good place to merge in your changes, and create the upstream PR from there?

That way we can incubate the feature in toip while waiting for it to go to DIF?

**Comment by darrellodonnell** [comment]
Would it make more sense to create a fork from the DIF SpecUp repo and do our work there? I am happy to do a bit of testing in there. @m00sey

**Comment by darrellodonnell** [comment]
I am happy to test the most recent version in the TRP (https://github.com/trustoverip/tswg-trust-registry-protocol) repo where I am quite active as well.

**Comment by m00sey** [comment]
> Would it make more sense to create a fork from the DIF SpecUp repo and do our work there? I am happy to do a bit of testing in there. @m00sey

That's what we did as part of the recent repo clean up and what was suggested in the spreadsheet.
The repo is here: https://github.com/trustoverip/spec-up

**Comment by brianorwhatever** [comment]
Superseded by #9

**Pull request closed without merging**